frame/staking/src/lib.rs

```diff
@@ -1576,19 +1653,30 @@ decl_module! {
 			}
 		}
 
-		/// The ideal number of validators.
-		#[weight = 5_000_000]
+		/// Sets the ideal number of validators.
+		///
+		/// The dispatch origin must be Root.
+		///
+		/// # <weight>
+		/// Base Weight: 1.717 µs
+		/// Write: Validator Count
+		/// # </weight>
+		#[weight = 2 * WEIGHT_PER_MICROS + T::DbWeight::get().writes(1)]
 		fn set_validator_count(origin, #[compact] new: u32) {
```

Comment:
this is not `DispatchClass::Operational`?

Reply:
similar question for others that require Root origin

Reply:
root origin != operational dispatch always.

That being said, I do think that we may want to re-evaluate some of the dispatches as being operational or not.

Probably another PR. But no assumptions should have changed here